### DISCUSSION TITLE
Where have you been traveling together with Little Backup Box?

### DISCUSSION THREAD

**Discussion by outdoorbits** [outdoorbits/little-backup-box#155] [General]
I would be very interested, where you and Little Backup Box traveled around. Would you give a quick or even better an extended report about your trips and your experience? Looking forward to your reports!

**Comment by Owz81** [comment]
After using some years a comeral sulution I found little backup box. So 3 week bevor the vacation, I odered parts, 3D-printed the case, build all together, installed the software, found a bug, got it fixed by outdoorbits, updated software, and all worked. 
I put all together in a small case, I took it with me every evening to lounch and make a copy of all during day used SD-Cards. We were 3 frineds to gether 3 weeks in Namibia. I saved pitures from 3 Cameras and one Gopro. The gopro was installed in our car and filming all the driven roads. So very evening I had up to 6h Video and up to 7000 Fotos to save. With the old comeral solution that would have take the hole night, with the backup box it was in worsed case 1,5h during dinner.
After I was back I just copyed the hdd to my NAS and send a link to both frinds so the can use all the pictues and videos made by any of use. 
It was the best update of my foto gear for vacation in the past years.

**Reply by outdoorbits** [reply]
Hi, very pleasant to read this, thank you! Did I get your right, you did all your backups using a cardreader? Could you please publish, which cardreader and what type of mass storage you used?

**Reply by Owz81** [reply]
Hi,
Cardreader: https://www.amazon.de/gp/product/B00LFIXC8I/ref=ppx_yo_dt_b_asin_title_o06_s00?ie=UTF8&psc=1
HDD: https://www.amazon.de/gp/product/B00KWHJVCY/ref=ppx_yo_dt_b_asin_title_o07_s00?ie=UTF8&psc=1

I also tested an other card reader from Delock but if I connected it, the harddisk stoppt working. My guess is the voltage dropped for a short time under the limit to keep the hdd turning and didn't start a again.

**Comment by ruebyi** [comment]
It was very useful on a family Vacation trip!
By Saving all files from iOS devices, different Cameras, GoPros, Drones etc. In one location it was easy to share them immediately when I was back home.

Import into XNView was quiet fast, because I had only to import from one SSD and not from 8 devices.

It‘s a huge Timesaver!

**Comment by rkosick** [comment]
I'm using Litte Backup Box since 13 months now on vacations in iceland, austria, norway, canarian islands, portugal, south korea, malaysia, borneo, australia and uganda. Always backup SD cards from different cameras (also from friends) to external SSD with a card reader. SD cards always formated to FAT or exFAT and external SSD formated to exFAT. For power supply I used different mobile powerbanks that have been all sufficient.

My setup:
Raspberry Pi Zero 2W with Zero4U 4-Port USB Hub for Raspberry Pi Zero V1.3&W (ZERO4U 2018 80)
Transcend USB 3.0 / 3.1 Gen 1 card reader TS-RDF5W: https://www.amazon.de/gp/product/B00FJRS8F8/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1
Samsung Portable T5 external SSD

Also started to use the cloud-services for backup to google drive. But was not happy with the upload speed. Now I configured the rsync server and its seems to be much faster. But I have to test it a few more times.

While being in hot environments, I had temperature problems with this setup backing up to cloud services. Reason for this were extra services I was running. For example I had a second wifi dongle connected that was creating another access point routing traffic via wireguard to my home network. This caused CPU utilization of almost 100%. After undervolting and limiting of CPU frequency the temperature problems were gone. But I'm not using additional stuff on it anymore anyway. 

Not using the image viewer and creation of thumbnails yet, so I don't know the performance of that on the raspberry pi zero 2W.

I never had any issues and I'm thankful for this great project. Before that I used my mobile phone with a USB hub. But this was so much slower and much more steps for every backup process.

Backup photos of the mobile phone includes another step since I have to backup first on a SD card. Thats because I have more pictures on the phone than I would like to backup. Since I take most of my pictures with digital cameras, thats not an issue for me.

PS: edit was adding 2W to raspberry pi zero in the text.

**Reply by outdoorbits** [reply]
Just changed "ZERO4U_2018_80" to "ZERO4U 2018 80" because search engines give better results for this.

**Comment by outdoorbits** [comment]
... and thank you very much for this fascinating report!

**Reply by rkosick** [reply]
I also tested my hardware setup using the image viewer and creation of thumbnails. It is pretty slow. 20 large jpeg images (about 5 MB each) need about 4 minutes. Have not tested RAW Files. More than 500 pictiures have been displayed with 2 hours. Transfer rate between the SSD as a target and SD card reader as source are 17 MB/s in average.

**Comment by outdoorbits** [comment]
For me, that sounds better than expected. It depends on your needs, but 
even 64 GB would be copied in about one hour - that would be about 2.000 
RAW images in my case. I never had a day where I had that many images to 
backup. Creating thumbnails is slow, for sure even much slower from 
RAW-images. But again, it depends on your needs. If you stay in a hotel 
with stable power supply - lbb can work while you sleep.

**Reply by rkosick** [reply]
Yes I'm really happy with the transfer speed. Doing it on my mobile phone was a real pain. Depending on how much pictures you take, thumbnails are also ok.

**Comment by qdmt** [comment]
![image](https://github.com/outdoorbits/little-backup-box/assets/124424854/4438f1d6-f531-4e1d-a636-832d5825ce69)

LBB has been great so far. Not only has it been helping add a little insurance policy during regular "plain vanilla" travels by having it do its thing back at the hotel, it's been a must have while overlanding expeditions off-road. Each night at camp, I'll have it run and leave the target storage separately while hiking or off-roading.

Photo of my setup. I'm not sure if the setup itself can be further optimized (can only get it to work if target {1TB NVME SSD} is on a separate power source), but it all fits in a pouch.

**Comment by outdoorbits** [comment]
I usually come back from traveling with a huge list of changes I want to make to lbb. That's why I usually don't write a post here, but get to work. Now this was different. Lbb did exactly what it was supposed to do all week and so I can now report back.
We have now been in Rome with the family for a week and have seen many impressive sights and learned a lot. If we had the time, we could certainly still admire the Eternal City for a few weeks.
The lbb was in the new housing and the display navigation proved to be very pleasant and direct, so I rarely used the web interface.
In addition to my Nikon Z6, we had five smartphones with us. I had previously configured and tested all of them accordingly at home. The images were saved on a SandDisk Extreme portable SSD 1 TB. Actually, I had another 2 TB SSD with me as a redundant backup. Thanks to a good upstream, the images were always at home the same evening via rsync, so there was no need for this local redundant backup.
I always travel with three sd cards with lbb installed and configured. On a previous trip I had "lost" one of these cards due to a power failure, lbb would no longer boot from this card. In Rome there were no such problems.
In the evening we sometimes looked at our pictures in fast forward via lbb View. However, we did not rate the pictures on the way - there was simply no time for that.

**Reply by qdmt** [reply]
👍👍👍

**Comment by AoXis** [comment]
Trying it in Tanzania this year.
Btw. it works with Sony a6400 and Fuji x70

**Comment by outdoorbits** [comment]
We just got back a week from our last trip. We spent three weeks backpacking with the kids in Thailand. We traveled with small luggage, also because we were traveling a lot and wanted to remain flexible. Once again, in addition to the Nikon Z6, several Android smartphones had to be secured. The GoPro Hero 10 was also used.
We only had minor technical problems with the GoPro. This only established a USB connection if the battery was briefly removed beforehand.
The internet connections were usually surprisingly fast. From most hostels we were able to upload the day's pictures to the local rsync server at night. We only had problems in one case when WiFi access could only be activated via a portal. A quick query with Dave Steele (Comitup) provided a solution for future trips. More about this in the [wiki](https://github.com/outdoorbits/little-backup-box/wiki/05b.-Comitup).
